### PR TITLE
sprites and shells

### PR DESCRIPTION
**Pull request by thedarkcrusader**
Replaces old 40mm grenade shell sprites with new ones.

Adds the FS Murphy,  SA Micic,
 FS Short Crcuit .

New teargas, Low yeild EMP, and White phosphorus  40mm shells

## About The Pull Request
This PR respite's and changes most of the 40mm grenades sprites with new ones. as long as adds in 3 new 40mm grenade shells Low range EMP.and Teargas shells  all of which will be obtainable via maints with the teargas shells being in the armory in a few boxes deleting some of the Stanground's  WPR's will be in the serbia kit

image

## Why It's Good For The Game
New sprites are great and more 40mm goodness to bring more flare to the table

## Testing
Tested every grenade shell to make sure they function properly everything works Code for the teargas shell works everything should be good to rock and roll 

## Changelog
:cl:
add: Adds new grenade sprites and sprites for some of the grenade code. thanks to kirov for sprites
del: Removed old  sprites
/:cl:

### CODE REVIEW

**Comment by thedarkcrusader** [comment]
Might be some grammatical errors this was done on an energetic deadlock so point them out if you see them

File: code/modules/projectiles/ammunition/bullets.dm [hunk 1]
```diff
@@ -484,18 +496,33 @@
 	matter = list(MATERIAL_STEEL = 3, MATERIAL_PLASTEEL = 2)
 	rarity_value = 25
 
+/obj/item/ammo_casing/grenade/frag/phosphorus
+	name = "SA WFR \"Micic\""
+	desc = "A military-grade White phosphorus grenade shell, designed to be fired from grenade launchers. The abbreviation \"SA\" marks its producer, but the lack of detatil makes it ubiquitous."
+	icon_state = "warcrime"
```
Comment (Firefox13) [inline code/modules/projectiles/ammunition/bullets.dm:502]: No.

Comment (Firefox13) [reply code/modules/projectiles/ammunition/bullets.dm:502]: Also, WFR?

Comment (thedarkcrusader) [reply code/modules/projectiles/ammunition/bullets.dm:502]: > No.

Yes

Comment (thedarkcrusader) [reply code/modules/projectiles/ammunition/bullets.dm:502]: > Also, WFR?

White phosphorus  round 
it should have been WPR but that was a typo

**Comment by thedarkcrusader** [comment]
Is the bot okay? I haven't  messed  with anything

**Comment by hyperioo** [comment]
The linter is complaining because of an unrelated issue in organ code, will be fixed once #8234 is merged.

**Comment by thedarkcrusader** [comment]
> 

Cool should be ready to rock  then

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -317,6 +317,15 @@
 	prespawned_content_amount = 6
 	prespawned_content_type = /obj/item/ammo_casing/grenade/frag
 
+/obj/item/storage/box/teargas_rounds
+	name = "box of pepperspray Shells"
+	desc = "A box containing 6 tear gas shells for use with a launcher. A gas mask is printed on the label.<br> WARNING: Exposure carries risk of serious injury or death. Keep away from persons with lung conditions."
```
Comment (hyperioo) [inline code/game/objects/items/weapons/storage/boxes.dm:322]:
```suggestion
	desc = "A box containing 6 tear gas shells for use with a launcher. A gas mask is printed on the label: " + span_bold("WARNING: Exposure carries risk of serious injury or death. Keep away from persons with lung conditions.")
```

File: code/modules/projectiles/ammunition/bullets.dm [hunk 3]
```diff
@@ -462,6 +462,18 @@
 	spawn_tags = SPAWN_TAG_AMMO
 	rarity_value = 15
 
+/obj/item/ammo_casing/grenade/teargas
+	name = "FS TGR \"Murphy\""
+	desc = "A Riot-grade Frozen Star teargas round, for use against unruly crowds."
```
Comment (hyperioo) [inline code/modules/projectiles/ammunition/bullets.dm:467]:
```suggestion
	desc = "A riot-grade Frozen Star tear gas round, for use against unruly crowds."
```

File: code/modules/projectiles/ammunition/bullets.dm [hunk 4]
```diff
@@ -491,11 +503,18 @@
 	projectile_type = /obj/item/projectile/bullet/grenade/emp // gonna keep this from being maint-lootable for now
 	matter = list(MATERIAL_STEEL = 3, MATERIAL_IRON = 2, MATERIAL_URANIUM = 2)
 
+/obj/item/ammo_casing/grenade/emp/low_yield
+	name = "FS EMPR \"Short Circuit\""
+	desc = "An EMP grenade shell, designed to be fired from grenade launchers. The abbreviation \"FS\" marks its producer, configured with a lower radius for use on ships and stations."
```
Comment (hyperioo) [inline code/modules/projectiles/ammunition/bullets.dm:508]:
```suggestion
	desc = "An EMP grenade shell configured with a lower radius for use on ships and stations, designed to be fired from grenade launchers. The abbreviation \"FS\" marks its producer,"
```

**Pull request merged**